bullet_train-api/app/helpers/api/open_api_helper.rb

```diff
@@ -46,6 +46,10 @@ def automatic_components_for(model, locals: {})
       path = "app/views/api/#{@version}"
       paths = [path, "app/views"] + gem_paths.product(%W[/#{path} /app/views]).map(&:join)
 
+      # Transform values the same way we do for Jbuilder templates
+      require "jbuilder/schema/template"
```

Comment:
@newstler, I think this `require` should be in the primary `schema.rb` file of the `jbuilder-schema` gem.

https://github.com/bullet-train-co/jbuilder-schema/blob/73a07d0d6c7f4628308f9f775ea4b9f486b8c38b/lib/jbuilder/schema.rb#L4

Unless there are reasons that people might want to use the gem but _not_ have the transformer immediately available for use.